array_experimental_test.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGetConsolidationPlan(t *testing.T) {
-	// Create an 1d array
+func createTestArray(t *testing.T) *Array {
+	// Create a 1d array
```

Comment:
since the function `createTestArray` would be visible to all tests in this package, it might be worth making the name more specific, since the array created may not be suitable for all tests. maybe `create1DTestArray`?

Reply:
Done